change/@ni-nimble-components-66a1a299-08a4-495c-9750-85902fe5413f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mollykreis @rajsite Did we reach a conclusion on whether the original use case which spurred this bug should actually be using nimble-toolbar (rather than the custom implementation they switched to)? If we think nimble-toolbar is appropriate, could you ping the original bug filer to let them know it's fixed (if you haven't already)?

Reply:
I don't think we reached an official conclusion, but my leaning is that they are probably better off with their custom implementation because the roving index of the nimble-toolbar won't work well with all the inputs they put in their toolbar. A user will probably have much better luck using the keyboard with their toolbar when each component is individually tabbable.

Reply:
Agreed, yea we probably need some usage guidance on nimble-toolbar for that and maybe higher-level patterns that discourage placing components with high interaction inside toolbars (framed around the aria guidance).